test/python/qpy/test_io_from_qpy.py

```diff
@@ -0,0 +1,33 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022, 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test cases for qpy loading and saving."""
+
+import io
+
+from qiskit import QuantumCircuit
+from qiskit.qpy import dump, load
+
+
+def test_io():
+    """Test QPy serialization and de-serialization."""
```

Comment:
It's not uncommon to refer to the issue in a regression test.
```suggestion
    """Test QPY serialization and de-serialization.
    See: https://github.com/Qiskit/qiskit/issues/11378"""
```

Reply:
Updated the docstring for the test, that now lives in `test/python/qpy/test_circuit_load_from_qpy.py`